src/main/java/ladder/domain/Point.java

```diff
@@ -3,25 +3,34 @@
 import java.util.Objects;
 
 public class Point {
+    private static final boolean CONNECTED = true;
+    private static final boolean NOT_CONNECTED = false;
```

Comment:
Point 객체 자체를 상수로 만들어두는 방법도 고려해 보시면 좋을 것 같아요 😃 
```suggestion
    private static final Point LEFT_POINT = new Point(Direction.LEFT);
    private static final Point RIGHT_POINT = new Point(Direction.RIGHT);
```